Datastore/src/Transaction.php

```diff
@@ -81,10 +81,10 @@ class Transaction
      * $transaction->commit();
      * ```
      *
-     * @param Entity $entity The entity to insert.
+     * @param EntityInterface $entity The entity to insert.
```

Comment:
https://github.com/jdpedrie/google-cloud-php/blob/1a85325587c1c962a995354fd3397c09b1e81358/Datastore/src/Transaction.php#L113
https://github.com/jdpedrie/google-cloud-php/blob/1a85325587c1c962a995354fd3397c09b1e81358/Datastore/src/Transaction.php#L181
https://github.com/jdpedrie/google-cloud-php/blob/1a85325587c1c962a995354fd3397c09b1e81358/Datastore/src/Transaction.php#L269

There are a few more instances of `Entity` -> `EntityInterface` that need to be hit.

Reply:
fixed.